selfdrive/car/honda/hondacan.py

```diff
@@ -94,14 +100,15 @@ def create_ui_commands(pcm_speed, hud, civic, accord, idx):
     commands.append(make_can_msg(0x39f, msg_0x39f, idx, 0))
   return commands
 
-def create_radar_commands(v_ego, civic, accord, idx):
+def create_radar_commands(v_ego, civic, accord, crv, idx):
```

Comment:
the signature will swell when more and more cars are supported. It's better to define a struct/class for the different type of cars.